runtime/cfdumper/main.c

```diff
@@ -1855,7 +1855,7 @@ static void dumpHelpText( J9PortLibrary *portLib, int argc, char **argv)
 	PORT_ACCESS_FROM_PORT(portLib);
 
 	vmDetailString(portLib, detailString, 1024);
-	j9file_printf( PORTLIB, J9PORT_TTY_OUT, "\nOpenJ9 Java(TM) Class File Reader, Version " EsVersionString);
+	j9file_printf( PORTLIB, J9PORT_TTY_OUT, "\nOpenJ9 Java(TM) Class File Reader, Version " JVM_VERSION_STRING);
```

Comment:
Should we put this into a `OPENJ9_` namespace?  I'm concerned this may conflict with a definition from OpenJDK

Reply:
How about `J9JVM_VERSION_STRING`? hopefully it applies to both `OpenJ9` and internal builds.